packages/esm-form-entry-app/src/app/loaders/json-loader.ts

```diff
@@ -0,0 +1,9 @@
+import { TranslateLoader } from '@ngx-translate/core';
+import { Observable, of } from 'rxjs';
+
+export class JsonLoader implements TranslateLoader {
+  constructor() {}
+  getTranslation(lang: string): Observable<any> {
+    return of(require(`../../../translations/${lang}.json`));
```

Comment:
```js
require.context(
  "../translations",
  false,
  /.json$/,
  "lazy"
);
```

Reply:
Actually, better suggestion for this is probably:

```suggestion
    return of(import(`../../../translations/${lang}.json`));
```

Reply:
Hi @ibacher!
I tried with the `import`, but the application didn't pick up the translations. 🤔